py5paisa/py5paisa.py

```diff
@@ -714,25 +723,31 @@ def get_oauth_session(self, request_token):
         except Exception as e:
             log_response(e)
 
-    def get_access_token(self, request_token):
+    def get_access_token(self, request_token=None):
```

Comment:
User do not have to pass None right ? just client.get_access_token() call will work ?